docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+       "Todo": 5
+     },
+     "models": {
+       "Todo": {
+         "1": "{\"title\":\"Take over the galaxy\",\"desc\":\"MWAHAHAHAHAHAHAHAHAHAHAHAHAMWAHAHAHAHAHAHAHAHAHAHAHAHA\",\"id\":1}",
+         "2": "{\"title\":\"destroy alderaan\",\"desc\":\"Make sure there are no survivors left!\",\"id\":2}",
+         "3": "{\"title\":\"play space invaders\",\"desc\":\"Become the very best!\",\"id\":3}",
+         "4": "{\"title\":\"crush rebel scum\",\"desc\":\"Every.Last.One.\",\"id\":4}"
+       }
+     }
+   }
+   ```
+
+### Add a Repository
+
+Let's a repository to handle the database CRUD functions of the `Todo` model
+using:
+
+```
+lb4 repository
+```
+
+You will be prompted with different questions. Specify the same responses as
+below.
+
+```
+? Please select the datasource DbDatasource
+? Select the model(s) you want to generate a repository Todo
+? Please select the repository base class DefaultCrudRepository (Legacy juggler bridge)
+   create src/repositories/todo.repository.ts
+   update src/repositories/index.ts
+
+Repository TodoRepository was created in src/repositories/
+```
+
+### Add a Controller
+
+Let's create a controller to define and handle the request-response lifecycle
+for our API endpoints using:
+
+```
+lb4 controller
+```
+
+You will be prompted with different questions. Specify the same responses as
+below.
+
+```
+? Controller class name: todo
+Controller Todo will be created in src/controllers/todo.controller.ts
+
+? What kind of controller would you like to generate? REST Controller with CRUD functions
+? What is the name of the model to use with this CRUD repository? Todo
+? What is the name of your CRUD repository? TodoRepository
+? What is the name of ID property? id
+? What is the type of your ID? number
+? Is the id omitted when creating a new instance? Yes
+? What is the base HTTP path name of the CRUD operations? /todos
+   create src/controllers/todo.controller.ts
+   update src/controllers/index.ts
+
+Controller Todo was created in src/controllers/
+```
+
+Look inside `src/controllers/todo.controller.ts`, and you will notice we have
+several more endpoints:
+
+- GET / POST / PATCH `/todos`
+- GET / PATCH / PUT / DEL `/todos/{id}`
+- GET `/todos/count`
+
+So, in total, these are the available endpoints of this application:
+
+- Application endpoint: http://localhost:3000/
+- API explorer: http://localhost:3000/explorer
+- Open API Spec: http://localhost:3000/openapi.json
+- API endpoints:
+  - http://localhost:3000/ping
+  - http://localhost:3000/todos
+  - http://localhost:3000/todos/{id}
+  - http://localhost:3000/todos/count
+- Health endpoint: http://localhost:3000/health
+- Liveness endpoint: http://localhost:3000/live
+- Readiness endpoint: http://localhost:3000/ready
+- Metrics endpoint: http://localhost:3000/metrics
+
+We will run, test, debug, and stop this application **locally** in the next
+section.
+
+## Running/Testing/Debugging/Stopping a LoopBack 4 Application Locally using Appsody
```

Comment:
We probably should mention the `watch` capability - as we make changes to the project locally, the application can be redeployed to a docker container as we go.

Reply:
> We probably should mention the `watch` capability - as we make changes to the project locally, the application can be redeployed to a docker container as we go.

@raymondfeng , I mention this in the sentence: `By default, `'File watching'` is **enabled** when you run the command
   `'appsody run'`. This means you can change a file in your project, and then
   Appsody builds/runs a new Docker container for you automatically.`